prod/westeurope/notifications/sandbox/storage_notifications/account/terragrunt.hcl

```
@@ -0,0 +1,24 @@
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_storage_account?ref=v3.0.3"
+}
+
+inputs = {
+  name                     = "notifications_sandbox"
```

```suggestion
  name                     = "notifsandbox"
```
1. _ not supported in storage name
2. storage name must be max 24 chars